incubator/cassandra/values.yaml

```diff
@@ -58,6 +58,6 @@ config:
 
 ## Configure node selector. Edit code below for adding selector to pods
 ## ref: https://kubernetes.io/docs/user-guide/node-selection/
-# selector:
-  # nodeSelector:
-    # cloud.google.com/gke-nodepool: pool-db
+## selector:
+  ## nodeSelector:
+    ## cloud.google.com/gke-nodepool: pool-db
```

Comment:
@Miouge1 This is failing lint because the comments are indented even though it's seen as being top level. If you update to something like this it should pass...

```
#  selector:
#    nodeSelector:
#      cloud.google.com/gke-nodepool: pool-db
```

Note, I used a single `#` because that's all that's needed for a YAML comment. One or two makes no difference to me.

Reply:
OK, I'll stick to the initial single `#`. It looks like text has `##` and content has `#`.